### PR TITLE
Chore/update with supabase demo deploy button

### PR DESCRIPTION
**Pull request by MildTomato**
### What?

- added correct deploy button urls
  - uses new supabase integration
- updated readme for local development, more clearer instructions
- added opengraph image

### Why?

- old supabase integration is buggy
- readme is more clear for local development

### CODE REVIEW

**Comment by MildTomato** [comment]
Also need @dijonmusters approval

File: examples/with-supabase/README.md [hunk 1]
```diff
@@ -1,24 +1,100 @@
...
-1. Run `npm install` to install dependencies
-1. Rename `.env.local.example` to `.env.local` and update the values for `NEXT_PUBLIC_SUPABASE_URL` and `NEXT_PUBLIC_SUPABASE_ANON_KEY` from [your Supabase project's API settings](https://app.supabase.com/project/_/settings/api)
-1. Run `npm run dev` to start the local development server
+3. Use `cd` to change into the app's directory
+
+    ```bash
+    cd name-of-new-app
+    ```
+
+4. Install the dependecies
```
Comment (dijonmusters) [inline examples/with-supabase/README.md:68]: This gets run for you by the `create-next-app` command 👍

File: examples/with-supabase/README.md [hunk 2]
```diff
@@ -1,24 +1,100 @@
...
+</p>
+<br/>
 
-The Vercel deployment will guide you through creating a Supabase account and project. After installation of the Supabase integration, all relevant environment variables will be set up so that the project is usable immediately after deployment 🚀
+## Features
 
-[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/clone?repository-url=https://github.com/vercel/next.js/tree/canary/examples/with-supabase&project-name=nextjs-with-supabase&repository-name=nextjs-with-supabase&integration-ids=oac_jUduyjQgOyzev1fjrW83NYOv)
+- [Next.js 13 ready](https://nextjs.org) App Router
+    - Client component examples
+    - React server component (RSCs) examples
```
Comment (dijonmusters) [inline examples/with-supabase/README.md:24]: Should be capitalised and pluralised as `Server Components`

**Review by dijonmusters — CHANGES_REQUESTED** [review]
Looks awesome! 🚀 Couple of small fixes

File: examples/with-supabase/README.md [hunk 3]
```diff
@@ -1,24 +1,100 @@
...
+  <a href="#how-to-use"><strong>How to use</strong></a> ·
+  <a href="#feedback-and-issues"><strong>Feedback and issues</strong></a>
+</p>
+<br/>
 
-The Vercel deployment will guide you through creating a Supabase account and project. After installation of the Supabase integration, all relevant environment variables will be set up so that the project is usable immediately after deployment 🚀
+## Features
 
-[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/clone?repository-url=https://github.com/vercel/next.js/tree/canary/examples/with-supabase&project-name=nextjs-with-supabase&repository-name=nextjs-with-supabase&integration-ids=oac_jUduyjQgOyzev1fjrW83NYOv)
+- [Next.js 13 ready](https://nextjs.org) App Router
```
[the remark this answers is not in the document]
Comment (MildTomato) [reply examples/with-supabase/README.md:22]: agree

[the remark this answers is not in the document]
Comment (MildTomato) [reply examples/with-supabase/README.md:22]: changed to app router

File: examples/with-supabase/README.md (hunk 1 above)
Comment (MildTomato) [reply examples/with-supabase/README.md:68]: removed

**Review by dijonmusters — APPROVED** [review]
LGTM! Thanks Jonny! 🙌

**Comment by MildTomato** [comment]
Wrong external-id used in deploy button. fixing

**Review by steven-tey — DISMISSED** [review]
LGTM! Thanks 🙏

**Review by leerob — COMMENTED** [review]
The `opengraph-image.png` file should be in `app/` and not `public`, right?

**Comment by MildTomato** [comment]
sorry @leerob - only just seen this, moved to `/app`

**Review by balazsorban44 — DISMISSED** [review]

**Pull request merged**